### PR TITLE
Add more specific enforcement via expected_column_types

### PR DESCRIPTION
**Pull request by vinceatbluelabs**
Here's a crack at the feedback re: expected_column_types - let me know what you think!

This is a PR back into the `mysql` branch - I'll take care of bigfiles over there to avoid a more complicated diff here.

### CODE REVIEW

File: tests/integration/records/table_validator.py [hunk 1]
```diff
@@ -112,7 +112,19 @@ def format_type(column: Dict[str, Any]) -> str:
             return str(column['type']) + suffix
 
         actual_column_types = [format_type(column) for column in columns]
-        assert actual_column_types in expected_column_types, actual_column_types
```
Comment (bluelabsbutler) [inline tests/integration/records/table_validator.py:115]: This file, one of the 3 largest in the project, increased in size to 423 lines.  The total size of those files is now 3465 lines (target: 897).  Is this file complex enough to refactor?

File: tests/integration/records/table_validator.py [hunk 2]
```diff
@@ -112,7 +112,24 @@ def format_type(column: Dict[str, Any]) -> str:
...
```
Comment (bluelabsbutler) [inline tests/integration/records/table_validator.py:115]: This file, one of the 3 largest in the project, increased in size to 428 lines.  The total size of those files is now 3470 lines (target: 897).  Is this file complex enough to refactor?

File: tests/integration/records/table_validator.py [hunk 3]
```diff
@@ -112,7 +112,29 @@ def format_type(column: Dict[str, Any]) -> str:
...
```
Comment (bluelabsbutler) [inline tests/integration/records/table_validator.py:115]: This file, one of the 3 largest in the project, increased in size to 433 lines.  The total size of those files is now 3475 lines (target: 897).  Is this file complex enough to refactor?

File: tests/integration/records/table_validator.py [hunk 4]
```diff
@@ -112,7 +112,30 @@ def format_type(column: Dict[str, Any]) -> str:
...
```
Comment (bluelabsbutler) [inline tests/integration/records/table_validator.py:115]: This file, one of the 3 largest in the project, increased in size to 434 lines.  The total size of those files is now 3476 lines (target: 897).  Is this file complex enough to refactor?

File: tests/integration/records/table_validator.py [hunk 5]
```diff
@@ -6,7 +6,11 @@
 from sqlalchemy.sql.elements import TextClause
 from typing import Optional, Dict, Any, Union, List
 from .timezone import set_session_tz
-from .expected_column_types import expected_column_types
```
Comment (bluelabsbutler) [inline tests/integration/records/table_validator.py:9]: This file, one of the 3 largest in the project, increased in size to 439 lines.  The total size of those files is now 3481 lines (target: 897).  Is this file complex enough to refactor?

**Comment by vinceatbluelabs** [comment]
Hi @cwegrzyn - could you review when you can?  This one is blocking the MySQL work.

**Pull request merged**